Sources/KlaviyoSwift/StateManagement.swift

```diff
@@ -144,11 +144,8 @@ struct KlaviyoReducer: ReducerProtocol {
             }
             state.initalizationState = .initializing
             state.apiKey = apiKey
-            // carry over pending events
```

Comment:
We are now doing this before completing initialization since we know that in certain cases requests may be added `state.pendingRequests` after this action has completed.